src/pages/courses/_layouts/CourseLayout.astro

```diff
@@ -73,10 +73,6 @@ const response = await fetch(
   `https://rockthejvm.com/api/purchase/${pricingPlanId}`,
 );
 
-if (!response.ok) {
```

Comment:
What was wrong with this?

Reply:
This triggers a failure on the entire page whenever the Teachable response doesn't arrive. Too aggressive, and fails in dev mode for all course pages. Kept the error just to the pricing card.